api/src/main/java/org/apache/iceberg/catalog/Catalog.java

```diff
@@ -303,4 +303,50 @@ default boolean dropTable(TableIdentifier identifier) {
    * @throws NoSuchTableException if the table does not exist
    */
   Table loadTable(TableIdentifier identifier);
+
+
+  /**
+   * Create a namespace in the catalog.
+   *
+   * @param namespace {@link Namespace}.
+   */
+  boolean createNamespace(Namespace namespace);
```

Comment:
I think this should be an optional trait. Tracking the existence of namespaces or namespace metadata is not a requirement of an Iceberg catalog.

Reply:
Why does this return a boolean?

Reply:
thanks Ryan , like spark interface , change the boolean to void